superset-frontend/src/dashboard/components/nativeFilters/FiltersConfigModal/FiltersConfigForm/ColumnSelect.tsx

```diff
@@ -128,6 +128,7 @@ export function ColumnSelect({
     <Select
       mode={mode}
       value={mode === 'multiple' ? value || [] : value}
+      ariaLabel="Column select"
```

Comment:
I think we could use the localization here

Reply:
Done